steno/src/model/stream/media_thread.cpp

```diff
@@ -68,7 +68,9 @@ void MediaThread::run()
     std::vector<Image> vcOutputFormatImages(1, Image(maxVcDim.width, maxVcDim.height, videoOutputConfig_->imageFormat));
 
     // TODO: config?
-    const int classifierRangeThreshold = 2;
+    const float classifierRangeThreshold = 0.35;
+
+    int chunkDurationMs = 1000 / videoInputConfig_->fpsTarget;
```

Comment:
If someone put 0 in the config it will crash. Maybe add a check and close the thread.

Reply:
plus add a print.

Reply:
should we throw an exception?

Reply:
An exception is a bit hard. We can just return and the thread will close. When we will have an error handling system, a message like "fps can't be zero" could shows up in the user's face.

    if (fps == 0) 
    {
        qCritical() << "target fps cannot be zero";
        return;
    }